sdk/js/src/token_bridge/getOriginalAsset.ts

```diff
@@ -76,20 +81,66 @@ export async function getOriginalAssetTerra(
   return getOriginalAssetCosmWasm(client, wrappedAddress, CHAIN_ID_TERRA);
 }
 
+/**
+ * Returns information about the asset
+ * @param wrappedAddress Address of the asset in question
+ * @returns Information about the asset
+ */
+export async function getOriginalAssetInjective(
+  wrappedAddress: string,
+  network: any
```

Comment:
Why does this take network instead of client?

Reply:
Fixed